### ISSUE TITLE
gqcpy tests/tutorials

### ISSUE THREAD

**Issue by lelemmen** [GQCG/GQCP#536]
**Describe the feature you'd like**
In #526, the old python-bound 'applications' were removed. Hence, we are currently in a state in which we do not test any of the gqcpy functionality.

Since I've already written some gqcpy tutorials, I think it would be a beneficial addition to the CI pipeline if all the gqcpy tutorials are up to date with any changes that have occurred in a new PR.

**Comment by guacke** [comment]
Reassigned to @GQCG/docs-developers.

**Comment by lelemmen** [comment]
In #583, I've added some example notebooks.

We can add this to our CI by using
```
jupyter nbconvert --to notebook --execute mynotebook.ipynb
```
as found in the [jupyter documentation](https://nbconvert.readthedocs.io/en/latest/execute_api.html).

**Comment by lelemmen** [comment]
After #590, it seems to be important that we use the `gqcp_dev` conda environment (in the `environment.yml`-file) in conjunction with this CI check.
